Support for CAP plugins (#717)
cap-js/docs: plugins/index.md

Following is a curated list of plugins that are available for the SAP Cloud Application Programming Model (CAP) which provide integration with SAP BTP services and technologies, or other SAP products.

::: tip Maintained by CAP and SAP
These plugins are created and maintained in close collaboration and shared ownership of CAP development teams and other SAP and BTP development teams.

> :::

Reviewer: I'd keep this tip to point out that all plugins on this page are CAP and SAP Plugins. Maybe we need to rephrase it for that purpose a little.

Author: the plugin page shows a selection (not all) cap-js and no cap-js-community plugins, is that the case? Then the tip is correct and we can leave it in

Reviewer: It shows cap-js and SAP-maintained cap-js-community plugins.

Author: e.g. cds-typer is in cap-js and not on the list (and I understand why as it has its own section in the docu already) but then it's not clear its included into the support process here (or is it not?)

   
## Support for Plugins

> The regular CAP support process does not apply for plugins. Issues are to be created in the issue page of the respective github repository.

Reviewer: This sentence is only valid for Node.js and can be found similarly in the following table.
```suggestion
The [regular CAP support process](/resources/#reporting-incidents) does not apply for plugins.
```

|    |    |
| --- | --- |

> | Node.js | Click the Node.js icon of the plugin in the list below to find the repository and open an issue there. 

Reviewer: When keeping the note above we can keep shorter here and avoid a linebreak:
<img width="298" alt="image" src="https://github.com/cap-js/docs/assets/32062331/91cbb0c8-0a3a-47e3-af30-bba7bfe65944">

```suggestion
| Node.js | Click the Node.js icon of the plugin to find the repository and open an issue there. 
```

Author: The reason I put this in was that reading the page from top to bottom it is not clear what item I should click (as I haven't seen the list below yet) - do you think it is clear enough without it? 
Is "Click the icon of the plugin in the list to find the repository and open an issue there." better?

One problem I see as well is that cap-js-community plugins are not on the list so there you'd need to find the repo in a different way.

Reviewer: I see your point. How about that?
```suggestion
| Node.js | Click the Node.js icon of the plugin in the list below and open an issue there.
```

Or:

```suggestion
| Node.js | Click the Node.js icon of the plugin in the list below and open an issue on GitHub.
```

Reviewer: > One problem I see as well is that cap-js-community plugins are not on the list so there you'd need to find the repo in a different way.

I guess we need to assume that users can make this transfer and find the plugins on GitHub. Therefore adding a link to community in the info note could be helpful as well.

| Java | Use the community to [ask a question](https://community.sap.com/t5/forums/postpage/board-id/technology-questions/interaction-style/qanda). 

<div id="internal-support" />

> Node.js plugins are available from two sources with different levels of support and maintenance:

Reviewer: ```suggestion
Node.js plugins are available with two different levels of support and maintenance:
```

Reviewer: Also avoiding the linebreak:
<img width="300" alt="image" src="https://github.com/cap-js/docs/assets/32062331/3811bfbc-667a-48e5-a85a-5337f6a8724c">

[CAP JS](https://github.com/cap-js) plugins are:
 - created and maintained in close collaboration and shared ownership of CAP development teams and other SAP and BTP development teams
 - supported like CAP itself

[CAP JS Community](https://github.com/cap-js-community) plugins are:
 - created and maintained by the community
 - not directly supported from CAP

:::info Complete list of plugins

> As CAP is blessed with an active community, there are many useful plugins available created by the community. Have a look at the two communities to browse all available plugins.

Reviewer: What other community? If there's more than one, we should add a link. Maybe we should add one anyway.

Reviewer: I think the goal should be to have all plugins from cap-js which are public (also in terms of repo=public) listed on this page.

I'll also discuss if we can include a link to https://bestofcapjs.org/ here. Needs just some legal clarification.
